### DISCUSSION TITLE
Reality vs. Results

### DISCUSSION THREAD

**Discussion by smros** [Ephigenia/ikea-availability-checker#95] [Q&A]
Hi!  I have been trying this out at our local store while we are waiting for all our kitchen stuff to arrive.  Today we picked up a few things,: one that was showing in on the script, but the others were all `0` all week, but managed to be in.

Am I doing something wrong with the Article numbers?  I am querying them simply as integers: ex. 702.656.76 is 70265676

Some examples show an "S" in front of the number, when is that requried?

-Steve

**Comment by Ephigenia** [comment]
> Hi! I have been trying this out at our local store while we are waiting for all our kitchen stuff to arrive. Today we picked up a few things,: one that was showing in on the script, but the others were all 0 all week, but managed to be in.

I’m sorry that the reality does not always cover what is actually is in store. You’re the first one that mentions such an issue. As the API is for real-world items there’s no guarantee that the numbers are always up to date for (I can only guess) a variety of reasons. 

> Am I doing something wrong with the Article numbers? I am querying them simply as integers: ex. 702.656.76 is 70265676

No, that is correct. The dots are removed when requesting the availability

> Some examples show an "S" in front of the number, when is that required?

Some IKEA product numbers have an "S" at the beginning for unknown reasons. The request for getting the availabilities is slightly different. This sofa for example has an "S" in front: https://www.ikea.com/de/de/p/soederhamn-ecksofa-3-sitzig-samsta-dunkelgrau-s89284463/
